Golden_Path.postman_collection.json

```diff
@@ -12661,11 +12696,577 @@
+									{
+										"key": "Date",
+										"value": "{{quoteDate}}"
+									},
+									{
+										"key": "FSPIOP-Source",
+										"value": "{{payerfsp}}"
+									},
+									{
+										"key": "FSPIOP-Destination",
+										"value": "{{payeefsp}}"
+									},
+									{
+										"key": "FSPIOP-Signature",
+										"value": "{{fspiop-signature}}",
+										"type": "text"
+									}
+								],
+								"body": {
+									"mode": "raw",
+									"raw": "{\n  \"quoteId\": \"{{quoteId}}\",\n  \"transactionId\": \"{{transactionId}}\",\n  \"payer\": {\n    \"partyIdInfo\": {\n      \"partyIdType\": \"MSISDN\",\n      \"partyIdentifier\": \"{{payerMSISDN}}\",\n      \"fspId\": \"payerfsp\"\n    },\n    \"personalInfo\": {\n      \"complexName\": {\n        \"firstName\": \"Mats\",\n        \"lastName\": \"Hagman\"\n      },\n      \"dateOfBirth\": \"1983-10-25\"\n    }\n  },\n  \"payee\": {\n    \"partyIdInfo\": {\n      \"partyIdType\": \"MSISDN\",\n      \"partyIdentifier\": \"22556999125\",\n      \"fspId\": \"payeefsp\"\n    }\n  },\n  \"amountType\": \"SEND\",\n  \"amount\": {\n    \"amount\": \"60.11\",\n    \"currency\": \"{{currency}}\"\n  },\n  \"transactionType\": {\n    \"scenario\": \"TRANSFER\",\n    \"initiator\": \"PAYER\",\n    \"initiatorType\": \"CONSUMER\"\n  },\n  \"note\": \"hej\"\n}"
+								},
+								"url": {
+									"raw": "{{HOST_QUOTING_SERVICE}}/quotes",
+									"host": [
+										"{{HOST_QUOTING_SERVICE}}"
+									],
+									"path": [
+										"quotes"
+									]
+								}
+							},
+							"response": []
+						}
+					],
+					"protocolProfileBehavior": {},
+					"_postman_isSubFolder": true
+				},
+				{
+					"name": "Tests for Bugfix #1009 - ML Adapter and ALS service health should include broker status",
+					"item": [
+						{
+							"name": "Get health status of ML Adapter",
+							"event": [
+								{
+									"listen": "test",
+									"script": {
+										"id": "bf62f6d3-8552-4b67-8519-310e28bb9614",
+										"exec": [
+											"pm.test(\"Status code is 200\", function () {",
+											"    pm.response.to.have.status(200);",
+											"});",
+											"",
+											"//Checking data",
+											"var jsonData = pm.response.json();",
+											"pm.test(\"Expected Status is OK\", function () {",
+											"   pm.expect(jsonData.status).to.eql(\"OK\");",
+											"",
+											"    pm.test(\"Expected body to have services array\", function () {",
+											"       pm.expect(jsonData.services).to.be.an('array');",
+											"       ",
+											"        pm.test(\"Expected services to have broker and status is OK\", function () {",
+											"            const brokerOk = jsonData.services.find(item => {",
+											"               return item.name=='broker' && item.status=='OK'; ",
+											"            });",
+											"            pm.expect(brokerOk).to.be.an('object');",
+											"   ",
+											"        });",
```

Comment:
Hey @vijayg10 i don't believe that the ml-api-adapter-service returns any services. so jsonData.status would be an empty array

Reply:
@rmothilal , Just checked and found that the recent ml-api-adapter-service does have broker health check.